src/scripts.js

```diff
@@ -54,23 +63,25 @@ window.addEventListener('load', () => {
     altFormat: "F j, Y",
     dateFormat: "Y/m/d"
   });
-  Promise.all([getCustomersData(), getBookingsData(), getRoomsData()])
+  Promise.all([getAllCustomersData(), getBookingsData(), getRoomsData(), getUserData('1')])
     .then(data => {
-      customers = data[0].customers
-      bookings = data[1].bookings
-      rooms = data[2].rooms
-      setDashboard(customers[1], bookings, rooms)
+      customers = data[0].customers;
+      bookings = data[1].bookings;
+      rooms = data[2].rooms;
+      user = data[3];
+      console.log(user);
```

Comment:
`console.log` that probably got forgotten/ left behind

Reply:
Good eye! Keeping this temporarily as a reminder to update user declarations once login page is functional.